rcgen/src/ring_like.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(feature = "ring")]
+pub(crate) use ring::*;
+
+#[cfg(all(not(feature = "ring"), feature = "aws_lc_rs"))]
+pub(crate) use aws_lc_rs::*;
+
+use crate::{error::ExternalError, Error};
```

Comment:
in the main Rustls crates we'd write these imports:

```suggestion
use crate::error::ExternalError;
use crate::Error;
```

Not blocking feedback here, just sharing in case you end up contributing to repos outside of this one in the Rustls org.

Reply:
Actually I like to know more about it. I never understood if and what is the general common convention for imports.

Personally (in my projects), I like to use `imports_granularity = "Item"` ([from cargo +nightly fmt](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=import#imports_granularity)) (an example [here](https://github.com/BiagioFesta/wtransport/blob/master/wtransport/src/config.rs#L35-L53)).

But that is just driven by my *personal* feeling of not having breaking lines and strange indentations in the import sections; I found it more.. "clean". However, I understand that sometimes is a waste of lines...

Reply:
There's a PR in the Rustls repo to automate the style: https://github.com/rustls/rustls/pull/1577

I don't know much about the history of how the style was arrived at but if you leave a comment there djc or ctz might have some thoughts !